core/components/TasksList/TasksList.tsx

```diff
@@ -67,21 +69,24 @@ const TasksList = ({
           />
         </TaskTitleWrapper>
       )}
-      <TasksLayout>
+
+      <TasksLayout data-testid={dataTestId}>
         {tasks?.map((task: TaskProps) => (
           <div key={task.id}>
             {task.id !== editTaskId ? (
-              <Task
-                setCurrentEdit={() => setEditTaskId(task.id)}
-                showCheck={listName !== TaskStatusType.COMPLETED ?? false}
-                showPause={[
-                  TaskStatusType.COMPLETED,
-                  TaskStatusType.CREATED,
-                ].includes(listName)}
-                task={task}
-                handleAction={(data: string) => handleAction(data, task)}
-                handleEdit={() => handleEdit(task)}
-              />
+              <div data-testid="task-item">
+                <Task
+                  setCurrentEdit={() => setEditTaskId(task.id)}
+                  showCheck={listName !== TaskStatusType.COMPLETED ?? false}
```

Comment:
false ?? 'Something' return false 
```suggestion
                  showCheck={listName !== TaskStatusType.COMPLETED}
```

Reply:
Removed, forgot to delete it.
It's was something else before this.